tutordiscovery/patches/local-docker-compose-dev-services

```diff
@@ -15,3 +15,12 @@ discovery:
     default:
       aliases:
         - "{{ DISCOVERY_HOST }}"
+
+{% if DISCOVERY_RUN_ELASTICSEARCH and is_docker_rootless() %}
+  elasticsearch:
+    ulimits:
+      memlock:
+        # Fixes error setting rlimits for ready process in rootless docker
+        soft: 0  # zero means "unset" in the memlock context
+        hard: 0
+  {% endif %}
```

Comment:
why is this needed?

Reply:
these settings are necessary for elasticsearch i think, when we are running docker in rootless mode to avoid memory issues for elasticsearch.